lang/values.md

```diff
@@ -97,7 +97,7 @@ Note, in particular, that `bool` just entirely ignored provenance; we discuss th
 ```rust
 impl Type {
     fn decode<M: Memory>(Type::Int(IntType { signed, size }): Self, bytes: List<AbstractByte<M::Provenance>>) -> Option<Value<M>> {
-        if bytes.len() != size.bytes() {
+        if size.bytes() != bytes.len() {
```

Comment:
again, operator overload ordering. `BigInt * usize` is defined, but `usize * BigInt` is not